_inc/client/at-a-glance/scan.jsx

```diff
@@ -184,54 +187,76 @@ class DashScan extends Component {
 		);
 	}
 
+	renderThreatsFound( numberOfThreats, dashboardUrl ) {
+		const { siteRawUrl } = this.props;
+		return (
+			<>
+				{ renderActiveCard( [
+					<h2 className="jp-dash-item__count is-alert">{ numberFormat( numberOfThreats ) }</h2>,
+					<p className="jp-dash-item__description">
+						{ __(
+							'Security threat found. Please {{a}}fix it{{/a}} as soon as possible.',
+							'Security threats found. Please {{a}}fix these{{/a}} as soon as possible.',
+							{
+								count: numberOfThreats,
+								components: {
+									a: <a href={ dashboardUrl } target="_blank" rel="noopener noreferrer" />,
+								},
+							}
+						) }
+					</p>,
+				] ) }
+				{ renderAction(
+					dashboardUrl,
+					__( 'View threat detail', 'View threats details', { count: numberOfThreats } )
```

Comment:
Github ate my previous comment on this.

This reads odd to me -- namely the "View threats details". I see we use `View security scan details` elsewhere, or maybe “View detailed results” or “view detailed result”?